README.md

```diff
@@ -5,41 +5,45 @@
 
  1. Open an Xcode Project that you would like to add real-time capabilities to.
  2. Select `File->Add Packages...`.
- 3. At the bottom of the Package Manager window, select `Add Local...`.
- 4. Select the directory containing the accompanying `Package.swift` and then select `Add Package`.
- 5. Under the `Link Binary with Libraries` build phase of your app target, add the `AirtimeMedia` library.
- 6. In order to publish through `LocalStream`'s, you need to provide a microphone usage description in your Info.plist
+ 3. In the search bar, put in the URL for this repository (https://github.com/airtimemedia/AirtimeMedia-iOS)
+ 4. Select the dependency rule that matches the version you want to use (i.e. if you want to use version 1.4.3, put in "Exact Version" as your dependency rule and enter 1.4.3 in the version box).
```

Comment:
For this to work we will need to remember to tag the merge commit as "1.4.3".  This should be automated for future releases.

Reply:
I looked into the automation but didn't find anything for creating automated GitHub releases. I guess we can just automate the tag creation.

Reply:
> Should we drop the "Early Access" from the README?
> 
> https://github.com/airtimemedia/AirtimeMedia-iOS/blob/8d3f140eb10ce3801a16eaa6e834497c2bfb691c/README.md?plain=1#L2

Yes